ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/deserialize/OrganisationMixIn.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.cmc.ccd.deserialize;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class OrganisationMixIn extends PartyMixIn {
+
+    @JsonProperty("partyContactPerson")
```

Comment:
IMHO `organisationContactPerson` and `organisationCompaniesHouseNumber` are more meaningful